src/explore-education-statistics-admin/src/pages/users/components/RoleForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { Role, User } from '@admin/services/userService';
+import Button from '@common/components/Button';
+import { FormFieldSelect, FormFieldset } from '@common/components/form';
+import Form from '@common/components/form/Form';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import Yup from '@common/validation/yup';
+import { Formik, FormikHelpers } from 'formik';
+import React from 'react';
+
+export interface UpdateRoleFormValues {
+  selectedRoleId: string;
+}
+
+interface Props {
+  roles: Role[] | undefined;
```

Comment:
Similar to the last comment I made you should be able to just do: 
`roles?: Role[]`

Reply:
I've changed this one too